exporter/src/lib.rs

```diff
@@ -0,0 +1,76 @@
+use std::path::PathBuf;
+use std::process::ExitCode;
+
+use guac::collector::emitter::NatsEmitter;
+use strum_macros::Display;
+
+pub mod exporter;
+
+#[derive(clap::ValueEnum, Debug, Clone)]
+pub enum Events {
+    #[clap(name = "kafka")]
+    Kafka,
+    #[clap(name = "sqs")]
+    Sqs,
+}
+
+#[derive(clap::ValueEnum, Debug, Clone, Display)]
+#[strum(serialize_all = "lowercase")]
+pub enum Storage {
+    #[clap(name = "bombastic")]
+    Bombastic,
+    #[clap(name = "vexation")]
+    Vexation,
+}
+
+#[derive(clap::Args, Debug)]
+#[command(about = "Run the exporter", args_conflicts_with_subcommands = true)]
+pub struct Run {
+    #[arg(short = 'i', long = "index")]
+    pub(crate) index: Option<PathBuf>,
+
+    #[arg(long = "kafka-bootstrap-servers", default_value = "localhost:9092")]
+    pub(crate) kafka_bootstrap_servers: String,
+
+    #[arg(long = "guac-url", default_value = "127.0.0.1:4222")]
+    pub(crate) guac_url: String,
+
+    // Event bus used to communicate with other services.
+    #[arg(long = "events", value_enum, default_value = "kafka")]
+    pub(crate) events: Events,
+
+    #[arg(long = "stored-topic", default_value = "sbom-stored")]
+    pub(crate) stored_topic: String,
+
+    #[arg(long = "devmode", default_value_t = false)]
+    pub(crate) devmode: bool,
+
+    #[arg(long = "storage", value_enum, default_value = None)]
+    pub(crate) storage: Storage,
```

Comment:
I think this parameter should be called 'bucket' because it configures the storage with the expected bucket to use, and it should be a string, because the bucket name can be anything the admin wants to use as the bucket  (even if it's hardcoded in some code now, which I'm working to address)

Reply:
How about we use the param 'storage-bucket'?

Reply:
I'll fix this in the refactoring